src/utils/configuration.ts

```diff
@@ -432,6 +437,21 @@ const isPlatformAuthEnabled = (): boolean => {
   return getCloudConfigurationValue('platformAuth', false)
 }
 
+const getDocsUrl = (path: string): string => {
+  const baseUrl = getCloudConfigurationValue('docsUrl', DEFAULT_DOCS_URL)
+  return `${baseUrl}#${path}`
+}
+
+const getBetaFlag = <T>(configName: keyof typeof configurationSchema.beta, defaultValue: T) => {
```

Comment:
@pastuxso why not reuse the experiments?

https://github.com/stateful/vscode-runme/blob/cris/unsupported-shell-warn/src/extension/kernel.ts#L138-L143